cmd/humioctl/filter_alerts_show.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2020 Humio Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/humio/cli/api"
+	"strings"
+
+	"github.com/humio/cli/cmd/internal/format"
+	"github.com/spf13/cobra"
+)
+
+func newFilterAlertsShowCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "show <repo-or-view> <name>",
```

Comment:
I think that all commands should be consistent in using `<view>` or `<repo-or-view>`.